app/src/main/java/com/example/grocerieswizard/shop/subshop/SubShoppingItem.java

```diff
@@ -1,13 +1,12 @@
-package com.example.grocerieswizard.models;
+package com.example.grocerieswizard.shop.subshop;
 
-import com.example.grocerieswizard.adapters.ShopAdapter;
+import com.example.grocerieswizard.shop.ShopAdapter;
 
 public class SubShoppingItem {
     private final String recipeName;
     private final String ingredientUnit;
     private final Double ingredientQuantity;
     private Boolean isChecked;
-    private ShopAdapter parentAdapter;
```

Comment:
Don't pass adapters, fragments, activites or interfaces in model classes. This should be as simple as possible. They should only hold the data, and maybe some small calculations.